sdks/go/pkg/beam/runners/prism/internal/stage.go

```diff
@@ -55,15 +55,16 @@ type link struct {
 // account, but all serialization boundaries remain since the pcollections
 // would continue to get serialized.
 type stage struct {
-	ID           string
-	transforms   []string
-	primaryInput string          // PCollection used as the parallel input.
-	outputs      []link          // PCollections that must escape this stage.
-	sideInputs   []engine.LinkID // Non-parallel input PCollections and their consumers
-	internalCols []string        // PCollections that escape. Used for precise coder sending.
-	envID        string
-	stateful     bool
-	hasTimers    []string
+	ID                   string
+	transforms           []string
+	primaryInput         string          // PCollection used as the parallel input.
+	outputs              []link          // PCollections that must escape this stage.
+	sideInputs           []engine.LinkID // Non-parallel input PCollections and their consumers
+	internalCols         []string        // PCollections that escape. Used for precise coder sending.
+	envID                string
+	stateful             bool
+	hasTimers            []string
+	processingTimeTimers map[string]bool
```

Comment:
Is https://github.com/lostluck/beam/blob/beam30083ProcessingTimme/sdks/go/pkg/beam/runners/prism/internal/engine/elementmanager.go#L843 the reason why we only have a map of processing time timers and not event time timers?

Reply:
There are only two time domains (Event, and Processing), and since event time was implemented first, it was the default, while processing time needs to be called out specifically.

There's certainly improvements we can make to how these are handled, but ultimately we need a mapping from timer names, to their domains in the element manager.